cet/README.md

```diff
@@ -0,0 +1,69 @@
+# Control-flow Enforcement Technology
+
+## Description
+Control-flow Enforcement Technology(CET) is a security feature in Intel
+Processors. The 11th generation and later Intel CPUs will support the CET
+feature.
+
+CET is the best method to prevent ROP/JOP attack from the root cause
```

Comment:
Is this really legitimate to say the "best method"? I don't know and just being conservative for this comment.

Reply:
Yes, it could prevent the ROP/JOP attack in instruction level.
Any other solution like software,  it needs more performance overhead, and low proteciton coverage.
So
"CET is the best method to prevent ROP/JOP attack from the root cause (instruction level) from compatibility and performance perspective.
"